viewer/package.json

```diff
@@ -81,11 +80,11 @@
     "webpack-dev-server": "^3.11.0",
     "webpack-log": "^3.0.1",
     "webpack-node-externals": "^2.5.1",
-    "whatwg-fetch": "^3.4.0",
-    "worker-plugin": "^5.0.0"
+    "whatwg-fetch": "^3.4.0"
   },
   "dependencies": {
     "@cognite/potree-core": "^1.1.3",
+    "@cognite/reveal-parser-worker": "link:../parser-worker",
```

Comment:
Before merging this PR I'm going to publish parser-worker as 1.0.0-beta-1 and change this line. Then merge this branch to master.